infra/modules/database/networking.tf

```diff
@@ -13,6 +13,14 @@ resource "aws_security_group" "role_manager" {
   vpc_id      = var.vpc_id
 }
 
+# We need to attach this security group to our DMS instance when created
+resource "aws_security_group" "dms" {
+  # checkov:skip= CKV2_AWS_5:DMS instance not created yet
```

Comment:
_(non-blocking comment)_ I'm interested in learning what this line is for!

Reply:
We use [checkov](https://www.checkov.io/1.Welcome/What%20is%20Checkov.html) as a tool to check our terraform files to make sure that they aren't misconfigured or configured in a way that could lead to problems down the line (e.g. bad OpSec). One of the policies that it enforces is that security groups need to be explicitly connected to a service, which I skipped because that's not the scope of this ticket